packages/server/api/controllers/products.controller.js

```diff
@@ -52,7 +53,26 @@ const getProductsByCategory = async (category) => {
   }
 };
 
+// Get Search product by Name
+const getProductBySearch = async (searchText, res) => {
+  if (!searchText) {
+    res.json([]);
+    return;
+  }
```

Comment:
This would more often be included under the '/' route and treat the search input as a query. 
If you want to keep it separate, it should not return an empty array for no search input. If it does, then you cannot just use this endpoint for the Collections page and you will have to combine it with the GET all products anyway - that is not very practical. 
In general, I don't see any good reason for this to be a separate endpoint, but maybe you do 😉

Reply:
I have created separate endpoint bcz I having difficulty to making swagger Api documentation now I find solution so I will add search query with category query.